test/test_ops.py

```diff
@@ -125,13 +131,162 @@ def test_inplace_gradgrad(self, device, dtype, op):
         self._gradgrad_test_helper(device, dtype, op, self._get_safe_inplace(op.get_inplace()))
 
 
-class TestOut(TestCase):
+class TestCommon(JitCommonTestCase):
```

Comment:
add comment here explaining class's purpose and why it inherits from JitCommonTestCase